sql/upgrade/sl3.0.sql

```diff
@@ -973,22 +994,16 @@ INSERT INTO invoice (id, trans_id, parts_id, description, qty, allocated,
             deliverydate, serialnumber
        FROM :slschema.invoice;
 
-ALTER TABLE :slschema.acc_trans ADD COLUMN lsmb_entry_id integer;
-
-update :slschema.acc_trans
-  set lsmb_entry_id = nextval('acc_trans_entry_id_seq');
-
 INSERT INTO acc_trans (entry_id, trans_id, chart_id, amount, transdate,
                        source, cleared, fx_transaction,
                        memo, approved, cleared_on, voucher_id, invoice_id)
- SELECT lsmb_entry_id, acc_trans.trans_id, (select id
-                    from account
-                   where accno = (select accno
-                                    from :slschema.chart
-                                   where chart.id = acc_trans.chart_id)),
-                                    amount, transdate, source,
-        CASE WHEN cleared IS NOT NULL THEN TRUE ELSE FALSE END, fx_transaction,
-        memo, approved, cleared, vr_id, invoice.id
+ SELECT lsmb_entry_id, acc_trans.trans_id,
```

Comment:
You stopped adding the `lsmb_entry_id` by removing the fragment above. Did you test this?

Reply:
Restored